core/slim_sim.cpp

```diff
@@ -4722,51 +4723,52 @@ void SLiMSim::ReorderIndividualTable(tsk_table_collection_t *p_tables, std::vect
 	}
 }
 
+
 void SLiMSim::SimplifyTreeSequence(void)
 {
 #if DEBUG
 	if (!recording_tree_)
 		EIDOS_TERMINATION << "ERROR (SLiMSim::SimplifyTreeSequence): (internal error) tree sequence recording method called with recording off." << EidosTerminate();
 #endif
-	
+
```

Comment:
please try to get rid of "noise" like this in the PR, it makes it hard to review